### ISSUE TITLE
geth data not persistent across container restart

### ISSUE THREAD

**Issue by LeeSmet** [threefoldfoundation/tfchain-templates#9]
restarting the container seems to trigger the consensus set to start syncing again from scratch

**Comment by LeeSmet** [comment]
```
In [84]: con.info
Out[84]: 
{'container': {'arguments': {'cgroups': [['devices', 'corex']],
   'config': {},
   'env': {'GETH_DATADIR': '/mnt/data',
    'GETH_LIGHTSERV': '90',
    'GETH_NAT': 'none',
    'GETH_NETWORK': 'testnet',
    'GETH_NODEKEY': '/sandbox/bin/bootnode.key',
    'GETH_PORT': '30302',
    'GETH_SYNCMODE': 'full',
    'GETH_V5DISC': 'v5disc',
    'GETH_VERBOSITY': '4'},
   'host_network': False,
   'hostname': '',
   'identity': '',
   'mount': {},
   'name': 'container-742863c0-4fe0-43d9-8b44-a0b6c0959c8d',
   'nics': [{'config': {'cidr': '', 'dhcp': False, 'dns': None, 'gateway': ''},
     'hwaddr': '',
     'id': 'None',
     'monitor': False,
     'name': 'nat0',
     'state': 'configured',
     'type': 'default'}],
   'port': {'30302': 30302},
   'privileged': False,
   'root': 'https://hub.grid.tf/tf-official-apps/geth.flist',
   'storage': 'zdb://hub.grid.tf:9900',
   'tags': ['container-742863c0-4fe0-43d9-8b44-a0b6c0959c8d']},
  'id': 2,
  'pid': 6427,
  'root': '/mnt/containers/2'}}
```

seems like the persistent dirs are not properly mounted

**Comment by LeeSmet** [comment]
Fixed in 0913b2d117447848b053fa9ba74cf5aa0fed8c5d
